app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorRequestView.java

```diff
@@ -296,20 +303,21 @@ private void enableUploadImageButtons(boolean setEnabled) {
   }
 
   /** Extracts favor data from and assigns it to currentFavor. */
-  private void getFavorFromView(Favor.Status status) {
+  private Favor getFavorFromView(Favor.Status status) {
```

Comment:
It doesn't matter to set return type as void or Favor

Reply:
ive changed this back to the original version